test/SourceKit/CursorInfo/cursor_implicit_init.swift

```diff
@@ -11,8 +11,12 @@ func foo() {
   MySubClass(abc:1)
 }
 
-// RUN: %sourcekitd-test -req=cursor -pos=10:12 %s -- %s | %FileCheck -check-prefix=CHECK1 %s
-// CHECK1: <Declaration>init(abc: <Type usr="s:Si">Int</Type>)</Declaration>
+// RUN: %sourcekitd-test -req=cursor -pos=10:7 %s -- %s | %FileCheck -check-prefix=CHECK1-INIT %s
```

Comment:
Why do you use column 7 here? Shouldn’t this be `3` if you want to refer to the `MyClass` token? Not that it makes much of a difference but it’s inconsistent with the rest of the tests. 

Similar below.